src/Tribe/I18n.php

```diff
@@ -190,13 +250,25 @@ public function compile_translations( array $strings, $domains ) {
 
 				// Grab the possible strings for default and any other domain.
 				if ( 'default' === $domain ) {
-					$strings[ $key ][] = __( $value );
-					$strings[ $key ][] = __( strtolower( $value ) );
-					$strings[ $key ][] = __( ucfirst( $value ) );
+					if ( $flags & static::COMPILE_INPUT ) {
+						$strings[ $key ][] = __( $value );
+					}
+					if ( $flags & static::COMPILE_STRTOLOWER ) {
+						$strings[ $key ][] = __( strtolower( $value ) );
+					}
+					if ( $flags & static::COMPILE_UCFIRST ) {
+						$strings[ $key ][] = __( ucfirst( $value ) );
```

Comment:
:no_entry_sign: The $text arg must be a single string literal, not "ucfirst( $value )".

<sub>hash: 3c69bb62cdc1ee2199053ac628e903d5</sub>